src/backend/oracle-data-interface/pom.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.6.4</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>ca.bc.gov.trafficcourtsonline</groupId>
```

Comment:
Is there a standard we should be following?  I see others have group id of `<groupId>ca.bc.gov.open</groupId>`

Reply:
I will change group id to "ca.bc.gov.open.jag" as well then.

Reply:
Great.  I noticed the main package of all the classes is still "ca.bc.gov.trafficcourtsonline.oracledatainterface".  Were you going to refactor the classes as well?
I might recommend "ca.bc.gov.open.jag.tco.oracledatainterface"

Reply:
Yeah makes sense. Refactored classes and packages as well.